Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -920,6 +927,23 @@ def addVertexFitting(
 
     customLogLevel = acts.examples.defaultLogging(s, logLevel)
 
+    selectedTrackParameters = "trackparameters"
+    trackIndices = "outputTrackIndices"
+
+    if trackSelectorConfig is not None:
+        trackSelector = acts.examples.TrackSelector(
+            level=customLogLevel(),
+            inputTrackParameters=trackParameters,
+            outputTrackParameters=selectedTrackParameters,
+            outputTrackIndices=trackIndices,
+            removeNeutral=True,
+            absEtaMax=trackSelectorConfig.absEtaMax,
+            ptMin=trackSelectorConfig.ptMin,
```

Comment:
```suggestion
            **acts.examples.defaultKWArgs(
               removeNeutral=trackSelectorRanges.removeNeutral,
               absEtaMax=trackSelectorRanges.absEta[1],
               ptMin=trackSelectorRanges.pt[0],
            ),
```
and add all the other parameters.